lib/reek/smells/nested_iterators.rb

```diff
@@ -44,7 +44,6 @@ def examine_context(ctx)
         else
           []
         end
-        # BUG: no longer reports nesting outside methods (eg. in Optparse)
```

Comment:
Boyscout rule. Stuff like that belongs into github issues and not in dangling comments.